File: Codeforces/Nearly_Lucky_Number.cpp
#include<iostream>
#include<string>
using namespace std;
int main()
{
    int temp=0,flag=0,counts=0;
string str;
cin>>str;
char ch;
for(int i=0;i<str.length();i++)
{
    ch=str.at(i);
    if(ch=='7'||ch=='4')
        temp++;
}
if(temp>0)
{
 if(temp==7||temp==4)
{
      cout<<"YES";
}
else
    cout<<"NO";
}
else
    cout<<"NO";



return 0;

}

File: Codeforces/NearlyLuckyNumber.java
import java.util.Scanner;
public class NearlyLuckyNumber
{
	public static void main(String asd[])
	{
		Scanner sc=new Scanner(System.in);
		String s=sc.nextLine();
		int temp=0;
		for(int i=0;i<s.length();i++)
			{
				if('7'!=s.charAt(i)||'4'!=s.charAt(i))
					temp++;
			}
		if(temp>1)
			System.out.println("NO");
		else
			System.out.println("YES");

		sc.close();
	}
}